pkg/services/guardian/accesscontrol_guardian.go

```diff
@@ -308,11 +449,67 @@ func (a *AccessControlDashboardGuardian) GetACL() ([]*dashboards.DashboardACLInf
 	return acl, nil
 }
 
-func (a *AccessControlDashboardGuardian) GetACLWithoutDuplicates() ([]*dashboards.DashboardACLInfoDTO, error) {
+// GetACL translate access control permissions to dashboard acl info
+func (a *accessControlFolderGuardian) GetACL() ([]*dashboards.DashboardACLInfoDTO, error) {
```

Comment:
no need to be back-ported but I would prefer to keep it because I have noticed that is used by the following endpoints:
`POST /dashboards/id/{DashboardID}/permissions`
`GET /folders/{folder_uid}/permission`
`POST /folders/{folder_uid}/permissions`

but it would be nice to get rid of `GetACL()` (note that if we manage to drop this we would no more need the [dashboard](https://github.com/grafana/grafana/blob/8dd3eb856de497b53b6065c533a181908286b6f1/pkg/services/guardian/accesscontrol_guardian.go#L130) field: the identifier and whether it's folder or not would be enough)